src/test/java/Homework20.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework20 extends BaseTest {
+
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+
+        //String uniquePlaylist = "1-HW19-Playlist";
+        String deletePlaylistMsg = "Deleted playlist \"1-HW19-Playlist.\"";
+
+        //Pre-condition
+        //Login
+        provideEmail("demo@class.com");
+        providePassword("te$t$tudent");
+        clickSubmit();
+
+        //Click on empty playlist created
+        clickOnPlaylist();
+
+        //delete selected playlist
+        clickOnDeleteBtn();
+
+        //Expected Results
+        WebElement notificationMsg = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.alertify-logs div.success")));//driver.findElement(By.cssSelector("div.alertify-logs div.success"));
+
+        Assert.assertEquals(deletePlaylistMsg,notificationMsg.getText());
```

Comment:
You got this problem, please correct it. Please finish your homework.

![image](https://github.com/Test-Pro-Bootcamp/A51-QA-Automation-Framework/assets/104761482/ed7e731b-5147-4d90-96e6-3e6c6979efca)